Src/EB/AMReX_EB_STL_utils.H

```diff
@@ -0,0 +1,42 @@
+#ifndef AMREX_EB_STL_UTILS_H_
+#define AMREX_EB_STL_UTILS_H_
+
+#include <AMReX.H>
+#include <AMReX_ParmParse.H>
+#include <AMReX_MultiFab.H>
+#include<AMReX_REAL.H>
+#include<AMReX_Vector.H>
+#include <AMReX_Geometry.H>
+#include <AMReX_FArrayBox.H>
+#include <AMReX_Box.H>
+
+using namespace amrex;
```

Comment:
Could you put everything into `namespace amrex {}` instead of `using namespace amrex` to avoid polluting the global namespace?

Reply:
everything is in amrex namespace now